src/app/shell/header/header-default/header-default.component.html

```diff
@@ -93,8 +93,8 @@
   </button>
 
   <div class="global-nav-container">
-    <nav class="global-nav navbar-expand-md" [ngbCollapse]="!showNavBar">
-      <div class="main-navigation" role="navigation">
+    <nav class="global-nav navbar-expand-md" [ngbCollapse]="!showNavBar" aria-label="main">
```

Comment:
the description of that specific nav-bar, since every nav-bar should have an aria-label that describes its role and this is the main nav-bar on the page it has the label "main" (has nothing to do with the role "main" because the role is defined by the `<nav>`-element